config/manager/controller_manager_config.yaml

```diff
@@ -9,3 +9,77 @@ webhook:
 leaderElection:
   leaderElect: true
```

Comment:
same comments apply here re: leaderElect and webhook

Reply:
done